### ISSUE TITLE
Add 'Lines of Code' to listing

### ISSUE THREAD

**Issue by ghost** [greensoftwarelab/Energy-Languages#14]
It would be interesting to compare program complexity (of which LOC is some sort of measure) to the energy and time usage.

**Comment by ScottPJones** [comment]
I agree, however, it would be good to measure only lines with code on them, not whitespace, documentation strings or just comments, to not penalize people who actually try to write readable code!

**Comment by chipbuster** [comment]
Given the nature of the benchmarks, I'm not sure that LOC is a meaningful metric for them. However, if anyone is interested in trying this, the [tokei](https://github.com/XAMPPRocky/tokei) tool can report total lines of code in various languages and differentiate between code, comments, and whitespace for most languages.
